src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -233,8 +233,7 @@ public static SafeFreeCredentials AcquireCredentialsHandle(SslAuthenticationOpti
                 if (newCredentialsRequested && sslAuthenticationOptions.CertificateContext != null)
                 {
                     SafeFreeCredential_SECURITY handle = (SafeFreeCredential_SECURITY)cred;
-                    // We need to create copy to avoid Disposal issue.
-                    handle.LocalCertificate = new X509Certificate2(sslAuthenticationOptions.CertificateContext.TargetCertificate);
+                    handle.LocalCertificate = true;
```

Comment:
It looks like this should be:
```suggestion
                    handle.HasLocalCertificate = true;
```

Reply:
Right, I accidentally ctrl-z' d the change